### PR TITLE
Show only important values in "display" select

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-designer/issues/261 https://github.com/webstudio-is/webstudio-designer/issues/603

- filtered to show only important values
- always show display as select instead of combobox for non-flex values

image

## Before requesting a review

- [ ] if the PR is WIP - use draft mode
- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")
- [ ] what kind of review is needed?
  - [x] conceptual
  - [x] detailed
  - [ ] with testing

## Test cases

- [ ] step by step interaction description and what is expected to happen

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

### CODE REVIEW

File: apps/designer/app/designer/features/style-panel/sections/layout/layout.tsx [hunk 1]
```diff
@@ -146,53 +130,72 @@ const LayoutSectionFlex = ({
...
+          />
+          <SelectControl
+            category={display.category}
+            currentStyle={display.currentStyle}
+            inheritedStyle={display.inheritedStyle}
+            setProperty={display.setProperty}
+            // show only important values first and hide others with scroll
+            styleConfig={{
+              ...display.styleConfig,
+              items: display.styleConfig.items
```
Comment (kof) [inline apps/designer/app/designer/features/style-panel/sections/layout/layout.tsx:176]: why do you need to use `display.styleConfig.items` at all if the list is fixed and already sorted?

Comment (TrySound) [reply apps/designer/app/designer/features/style-panel/sections/layout/layout.tsx:176]: Labels

File: apps/designer/app/designer/features/style-panel/sections/layout/layout.tsx [hunk 2]
```diff
@@ -146,53 +130,72 @@ const LayoutSectionFlex = ({
   );
 };
 
+const orderDisplayValuesFirst = [
```
Comment (kof) [inline apps/designer/app/designer/features/style-panel/sections/layout/layout.tsx:133]:
```suggestion
const orderedDisplayValues = [
```

**Review by kof — APPROVED** [review]
nit

**Pull request merged**